input/pages/measure-conformance.md

```diff
@@ -70,7 +70,7 @@ Snippet 3-4 illustrates a FHIR Library resource containing a CQL library with a
 **Conformance Requirement 3.1 (Referencing CQL Documents):** [<img src="conformance.png" width="20" class="self-link" height="20"/>](#conformance-requirement-3-1)
 {: #conformance-requirement-3-1}
 
-1. FHIR-based eCQMs SHALL consist of a FHIR Measure resource conforming to at least the CRMIShareableMeasure profile. In addition, measures with a status of active SHALL conform to the CQFMPublishableMeasure profile In particular, FHIR-based measures SHALL contain a narrative containing a human-readable representation of the measure content.
+1. FHIR-based eCQMs SHALL consist of a FHIR Measure resource conforming to at least the [CRMIShareableMeasure profile](http://hl7.org/fhir/uv/crmi/STU1/StructureDefinition-crmi-shareablemeasure.html). In addition, measures with a status of active SHALL conform to the [CQFMPublishableMeasure profile](http://hl7.org/fhir/uv/crmi/STU1/StructureDefinition-crmi-publishablemeasure.html). In particular, FHIR-based measures SHALL contain a narrative containing a human-readable representation of the measure content.
```

Comment:
"In particular, " conflicts with the changes applied in https://github.com/HL7/cqf-measures/pull/247 for the run-on sentence. Needs to correct it so it is consistent with PR 247. 

"..... measures with a status of active SHALL conform to the CQFMPublishableMeasure profile in particular. ...." instead of ". In particular, ..."